state/futures/actions.ts

```diff
@@ -1 +1,38 @@
-export default null;
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+import { Period } from 'sdk/constants/period';
+import { FuturesMarketSerialized } from 'sdk/types/futures';
+import { AppThunk } from 'state/store';
+import { ThunkConfig } from 'state/types';
+import { FuturesMarketAsset, serializeMarkets } from 'utils/futures';
+
+import { FundingRateSerialized } from './types';
+
+export const setMarketAsset = (asset: FuturesMarketAsset): AppThunk => (dispatch) => {
```

Comment:
Since this does not have any async requirements, I was wondering if there was a specific reason we don't just use the action returned from `createSlice` directly.

Reply:
Yea good spot, think it was in anticipation and then never added anything, will update.